front/app/components/UI/ClaveUnicaButton/index.tsx

```diff
@@ -1,46 +1,76 @@
 import React from 'react';
 
-import { TVerificationMethod } from 'api/verification_methods/types';
+import { Icon } from '@citizenlab/cl2-component-library';
+import styled from 'styled-components';
```

Comment:
Bit hard to understand these changes, but basically this was `UI/ClaveUnicaButton/ClaveUnicaButton`, and then `UI/ClaveUnicaButton/index` was a wrapper around this. But the wrapper made 0 sense because it contained logic that was only relevance for the verification button. So instead I moved the wrapper to the `id_clave_unica` module where it belongs